ds3-autogen-c/src/test/java/com/spectralogic/ds3autogen/c/RequestConverter_Test.java

```diff
@@ -53,15 +62,19 @@ public void testEmptyResponsePayload() {
     }
 
     @Test
-    public void testGetRequestParameterListNoResponseType() {
-        final ImmutableList<Parameter> paramList = RequestConverter.getParamList("");
+    public void testGetRequestParameterListNoResponseType() throws IOException {
+        final Ds3DocSpecParser docSpecParser = new Ds3DocSpecParserImpl(new NameMapper());
+        final Ds3DocSpec docSpec = docSpecParser.getDocSpec(CCodeGenerator_Test.class.getResourceAsStream("/input/testCommandDocumentation.json"));
+        final ImmutableList<Parameter> paramList = RequestConverter.getParamList("", docSpec);
```

Comment:
Using an empty doc spec here would be simpler
`... = RequestConverter.getParamList("", new Ds3DocSpecEmptyImpl());`